addons/mail/static/src/models/activity/activity.js

```diff
@@ -234,6 +234,19 @@ function factory(dependencies) {
         _computeMessaging() {
             return [['link', this.env.messaging]];
         }
+
+        /**
+         * Wysiwyg editor put `<p><br></p>` event if the field note is empty. This compute ignore this empty value.
```

Comment:
```suggestion
         * Wysiwyg editor put `<p><br></p>` even if the field note is empty. This compute replaces this almost empty value by an actual empty value, to reduce the size the empty note takes on the UI.
```
Typo + add the "why". You should appropriately split it in multi-line comment.

Reply:
Done